threader3000.py

```diff
@@ -30,8 +31,11 @@ def main():
     print("                   A project by The Mayor               ")
     print("-" * 60)
     time.sleep(1)
-    target = input("Enter your target IP address or URL here: ")
```

Comment:
No changes will be made to this functionality at this time with address/URL capture, but thank you for the suggestion here.